molgenis-apps/src/main/java/org/molgenis/apps/model/AppMetaData.java

```diff
@@ -53,11 +53,13 @@ protected void init()
 		addAttribute(IS_ACTIVE).setDataType(BOOL).setLabel("Active").setNillable(false).setVisible(false)
 				.setDefaultValue(Boolean.FALSE.toString());
 		addAttribute(USE_FREEMARKER_TEMPLATE).setDataType(BOOL).setLabel("Use freemarker template as index.html")
-				.setNillable(false).setDefaultValue(Boolean.TRUE.toString());
+				.setNillable(false).setDefaultValue(Boolean.FALSE.toString()).setDescription(
+				"When using a freemarker template, the index page of your app should be placed in "
```

Comment:
That would be nice yes, however I don't see any way to implement that given the current functionality of the app meta data. I cannot set a description conditionally.